.github/workflows/Publish-Website.yml

```diff
@@ -29,7 +29,7 @@ jobs:
           java-version: 11.0.11+9
 
       - name: Generate API docs
-        run: ./gradlew dokkaGfm
+        run: gradle dokkaGfm
```

Comment:
How does this work (i.e. where is Hermit configured?)

Reply:
Good point.  I was partially trying this in the github builds to see what happens.